src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/FontRendererAccessor.java

```diff
@@ -0,0 +1,16 @@
+package com.mitchej123.hodgepodge.mixins.early.minecraft;
+
+import net.minecraft.client.gui.FontRenderer;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+@Mixin(FontRenderer.class)
+public interface FontRendererAccessor {
+
+    @Invoker("getFormatFromString")
+    static String callGetFormatFromString(String s) {
```

Comment:
This is not what I meant with "Access Transformer"

Reply:
Can you explain what you want then ? Cause that's how you access vanilla methods : with an invoker

Reply:
Initially
- You create a file called `hodgepodge_at.cfg` in the `src/main/resources/META-INF` directory.
- You set the the `accessTransformersFile` property in `gradle.properties`.

To actually use ATs
1. You add a line in that `hodgepodge_at.cfg` file according to the [Forge wiki](https://docs.minecraftforge.net/en/latest/advanced/accesstransformers/).
2. You recompile the MC sources using `gradlew build` / `gradlew assemble` / `gradlew sDecW`.
3. You reload the project in your IDE.
4. You can access the transformed method/field directly in your code.

Reply:
Example: https://github.com/GTNewHorizons/NotEnoughItems/blob/master/src/main/resources/META-INF/nei_at.cfg

Reply:
I'm not going to use forge's access transformers.

Reply:
@mitchej123 we disagree, I want to use mixin invoker to access the method, they want me to use forge access transformers, since it's your repo which one do you prefer ?

Reply:
I prefer mixin accessors or reflection over access transformers in general.